internal/store/postgres/job/job_repository.go

```diff
@@ -120,6 +120,31 @@ WHERE project_name = $4 AND namespace_name = $5 AND name = any ($3)
 	return nil
 }
 
+func (j JobRepository) SyncState(ctx context.Context, jobTenant tenant.Tenant, disabledJobs, enabledJobs []*job.Name) error {
+	tx, err := j.db.Begin(ctx)
+	if err != nil {
+		return err
+	}
+	updateJobStateQuery := `
```

Comment:
should we also update the `updated_at` field?

Reply:
since this will be updated regularly from a scheduled dag, this will reset the actual job update event time

Reply:
got that